requirements.txt

```diff
@@ -10,7 +9,6 @@ dask==2022.8.0
 deepdiff==5.8.0
 emails
 fastapi[all]==0.81.0
-fastapi-caching[redis]
```

Comment:
I also moved this to `optional` under the `redis` subheading, I'm assuming this is correct? This is only needed for Redis caching or without Redis are we still using this package in some way?